Adding a new tip to discover all events fired in one request (#123)
LaravelDaily/laravel-tips: log-and-debug.md

Tip given by [@mmartin_joo](https://twitter.com/mmartin_joo/status/1473262634405449730)

### Discover all events fired in one request

> If you want to implement a new listener to especific event but you dont know what is your name, u can log all events fired during the request.

Reviewer: ```suggestion
If you want to implement a new listener to a specific event but you don't know its name, you can log all events fired during the request.
```

You can use the `\Illuminate\Support\Facades\Event::listen()` method on `boot()` method of `app/Providers/EventServiceProvider.php` to catch all events fired.

> > Important: Do not use the Log::* methods to not make a infinite loop (or skip all Log::* events)

Reviewer: ```suggestion
**Important:** If you use the `Log` facade within this event listener then you will need to exclude events named `Illuminate\Log\Events\MessageLogged` to avoid an infinite loop. 
(Example: `if ($event == 'Illuminate\\Log\\Events\\MessageLogged') return;`)
```

```php

> public function boot()

Reviewer: ```suggestion
// Include Event...
use Illuminate\Support\Facades\Event;

// In your EventServiceProvider class...
public function boot()
```
